src/simplewallet/simplewallet.cpp

```diff
@@ -164,7 +167,7 @@ static std::string get_human_readable_timespan(uint64_t seconds);
 
 namespace
 {
-  const std::array<const char* const, 5> allowed_priority_strings = {{"default", "unimportant", "normal", "elevated", "priority"}};
+  const std::array<std::string_view, 5> allowed_priority_strings = tools::fee_priority_utilities::feePriorityStrings_;
```

Comment:
Its nitpick, but there's still a bunch of camel case in this patch. Typically everything is snake case.

Reply:
See comment below: https://github.com/monero-project/monero/pull/9838#issuecomment-2972073678 Done